.gitignore

```diff
@@ -5,3 +5,5 @@ target
 custom_checkstyle.xml
 
 liquibase.properties
+
+/?
```

Comment:
With the correction to setting the relevant arguments and env vars for the maven home folder `Users/1000` this ignore shouldn't be needed as it was the volume mapping in conjunction with how maven was previously called that was causing the folder '?' to appear